test/java-builder-tests/src/main/resources/profiles/basic/environment.properties

```diff
@@ -16,6 +16,8 @@
 #
 
 
+test.service.endpoints=http://localhost:8080
```

Comment:
we agreed to leave env.properties for launching framework (in stand by mode). All parameters, related to the test execution are the part of test properties now. This particular case is a good chance to verify our new approach ;)